resources/assets/js/components/GroupDeviceRepairPodium.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div>
+    <div class="d-none d-md-flex flex-column justify-content-end height">
+      <div :class="'medal-' + position">
+        <div class="bg-white d-flex flex-column justify-content-between text-center font-weight-bold medal">
+          <div>
+            <b-img-lazy class="position-absolute rosette shift" :src="'/images/rosette_' + position + '_ico.svg'" />
```

Comment:
Yes, it does, and this can decrease your page load time so it's good to use it.  There's a slight difference in error handling, I think, but otherwise it's just exactly the same.